src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,170 @@
 package core.basesyntax;
 
+//import jdk.internal.util.ArraysSupport;
+import java.util.Arrays;
+import java.util.NoSuchElementException;
+
+
 public class ArrayList<T> implements List<T> {
+
+
```

Comment:
Please, remove not needed whitespace between method and class

https://mate-academy.github.io/style-guides/java/java.html#s4.6.1-vertical-whitespace